.buildkite/scout_ci_config.yml

```diff
@@ -1,6 +1,8 @@
 # Define which plugins should be run or skipped in Scout CI pipeline
 ui_tests:
   enabled:
+    - slo
+  disabled:
```

Comment:
@dmlemeshko This is just for testing? Let's not forget to re-enable the plugins.